tee-worker/ts-tests/integration-tests/common/utils/assertion.ts

```diff
@@ -398,3 +403,79 @@ export function assertIdentityLinkedResult(
         parseIdentity(context.sidechainRegistry, decodedLinkResult.identity, aesKey).toString()
     );
 }
+/* 
+    assert vc
+    steps:
+    1. compare vc account with signer
```

Comment:
I think this is bad assumption.
The response `account` field holds information about `subject` not `signer`.

Reply:
you mean `compare vc account with subject`?

Reply:
yes

Reply:
It's left untouched. You modified other function doc